heron/api/src/java/BUILD

```diff
@@ -18,16 +18,16 @@ java_library(
 )
 
 java_binary(
-    name = "api-unshaded",
+    name = "api",
     srcs = glob(["com/twitter/heron/api/**/*.java"]),
     deps = api_deps_files,
 )
 
-genrule(
+jarjar_binary(
     name = "heron-api",
-    srcs = [":api-unshaded_deploy.jar"],
-    outs = ["heron-api.jar"],
-    cmd  = "cp $< $@",
+    src = ":api_deploy.jar",
+    shade = "shade.conf",
```

Comment:
I see. I missed that we were only shading com.google. I thought all of heron-api was affected.